samples/bluetooth/fast_pair/locator_tag/sysbuild/b0n/prj.conf

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Based on nrf/samples/nrf5340/netboot/prj.conf and
+#          nrf/samples/nrf5340/netboot/overlay-minimal-size.conf
```

Comment:
I wonder if we could pass the `overlay-minimal-size.conf` to `b0n` image in similar way as we do it for buffer size increase in netcore image (to avoid duplicating configuration)